i18n/en.json

```diff
@@ -154,6 +154,7 @@
   "I18N_PLAYER_COMMUNITY_EDITABLE_TOOLTIP": "Community editable",
   "I18N_PLAYER_CONTINUE_BUTTON": "Continue",
   "I18N_PLAYER_CONTINUE_NEXT_CARD": "Continue to next card",
+  "I18N_PLAYER_SHARE_EXPLORATION" : "Enjoyed this exploration? Share it with your friends!",
```

Comment:
Great!

On Mon, Jul 4, 2016 at 12:08 AM, Sean Lip <notifications@github.com> wrote:

> In i18n/en.json
> <https://github.com/oppia/oppia/pull/2212#discussion_r69398606>:
>
> > @@ -154,6 +154,7 @@
> >    "I18N_PLAYER_COMMUNITY_EDITABLE_TOOLTIP": "Community editable",
> >    "I18N_PLAYER_CONTINUE_BUTTON": "Continue",
> >    "I18N_PLAYER_CONTINUE_NEXT_CARD": "Continue to next card",
> > +  "I18N_PLAYER_SHARE_EXPLORATION" : "Enjoyed this exploration? Share it with your friends!",
>
> Great, thanks @anafrancisca <https://github.com/anafrancisca>!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/oppia/oppia/pull/2212/files/8e799bb546371c56206888362ef61f3f5644f065#r69398606>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARewAitmfszSujrk--rSH-JIm6Ok_Fwbks5qSEDRgaJpZM4JD9xP>
> .
>



-- 
Ana Francisca Bernardo

https://pt.linkedin.com/pub/ana-francisca-bernardo/9b/7a2/a63

https://fcsh-unl.academia.edu/AnaFranciscaBernardo